CovidPolicy.md

```diff
@@ -0,0 +1,43 @@
+# PyBay Attendance Requirements
+
+## Policy Overview
+
+###Vaccine or Negative COVID-19 Test Expectation
+
+1.     [**Outdoor Event Guidelines**](#outdoor-event-guidelines)
+1.     [**Vaccine Expectation**](#vaccine-expectation)
+1.     [**Negative COVID-19 Test Expectation**](#negative-covid-19-test-expectation)
+
+### Outdoor Event Guidelines
+
+In line with San Francisco guidelines, we prioritize the well-being of our attendees at this outdoor event. While masks and vaccinations are not currently mandated, we reserve the right to change this at any time to comply with San Francisco guidelines. Refunds will be offered for attendees who are unable to comply with updated guidelines. At this time, personal choices regarding their usage are respected. We encourage everyone to uphold good hygiene practices, maintain social awareness, and consider vaccination for the safety of all. Should you feel unwell or exhibit symptoms, we kindly request staying home. Your health and enjoyment are paramount, and we're committed to fostering a secure environment throughout the event. We will refund your ticket if you are unwell.
+
+We will have masks available at the conference for anyone who needs them. It is also expected that requests for personal space will be respected by attendees and are covered under the code of conduct. 
```

Comment:
Maybe we can break this into two parts?

"It is expected that requests for personal space will be respected by attendees. Respecting such requests is covered by the PyBay code of conduct."

Reply:
agreed.